management/user_test.go

```diff
@@ -465,3 +466,22 @@ func TestUserIdentity(t *testing.T) {
 		}
 	})
 }
+
+func givenAUser(t *testing.T) *User {
+	user := &User{
+		Connection: auth0.String("Username-Password-Authentication"),
+		Email:      auth0.String("chuck@chucknorris.com"),
```

Comment:
```suggestion
		Email:      auth0.String("chuck@example.com"),
```
Replace the domain name with a IANA-reserved domain name (https://www.rfc-editor.org/rfc/rfc6761.html#section-6.5) as chucknorris.com is an actual domain.